index.css

```
@@ -0,0 +1,58 @@
+:root {
+    --active-border-color: #f0da50;
+    --bg-color: #f4f4f4;
+    --text-color: #6d6d6d;
+    --tab-bg-color: #ffffff;
```

Краще може використовувати скорочений запис у випадках, коли це можливо?
```suggestion
    --tab-bg-color: #fff;
```

---

Загалом СSS-проперті — ❤️.